UI/admin.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta charset="utf-8">
+	<title>User Profile | iReporter</title>
+	<link rel="stylesheet" type="text/css" href="style/main.css">
+</head>
+
+<body>
+	<header>
+		<div class="container">
+			<div id="title">
+				<h1><span class="logo">iReporter</span> Admin</h1>
+			</div>
+			<nav>
+				<ul>
+					<li><a href="admin.html">Administrator</a></li>
+					<li><a href="login.html">Log out</a></li>
+				</ul>
+			</nav>
+		</div>
+	</header>
+	<main>
+		<section id="welcome">
+			<div class="container">
+				<h3>Welcome Admin!</h3>
+		</section>
+		<section id="record-status">
+			<div class="container">
+				<div class="box">
+					<a href="#">
+						<h1>148</h1>
+						<p>records has been resolved.</p>
+					</a>
+				</div>
+				<div class="box">
+					<a href="#">
+						<h1>93</h1>
+						<p>records are under investigation</p>
+					</a>
+				</div>
+				<a href="#">
+					<div class="box">
+						<h1>48</h1>
+						<p>records were rejected.</p>
+				</a>
+			</div>
+			</div>
+			<div class="see-all">
+				<h3>Analyze user records... <a href="#">Click here</a></h3>
+			</div>
+			</div>
+		</section>
+		<footer>
+			<p>iReporter, Copyright &copy; 2018</p>
+		</footer>
+</body>
+
+</html>
```

Comment:
Indent spaces here are too much. 2 or 4 is sufficient. Look into an editor config file for your editor

Reply:
Also No new line at End of File